jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -228,6 +228,50 @@
                   (send bbox :header :frame_id)
                   (send bbox :pose)))
       bx))
+  (:cube->cube-parallel-to-coords
+    (cube coords &rest args)
+    (let (copycoords target-ids cube-min-max-list)
+      (send cube :worldcoords)
+      ;; without this, body isn't updated
```

Comment:
@knorth55 This is also needed in https://github.com/start-jsk/jsk_apc/blob/stow-task/jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l#L276-286
After you call `:worldcoords`, `cube-pos` isn't needed.